.circleci/config.yml

```diff
@@ -15,5 +15,6 @@ workflows:
       - path-filtering/filter:
           name: check-for-changes
           mapping: |
-            (model/.*|hmpps-prisoner-search/.*) workflow-directory "hmpps-prisoner-search"
```

Comment:
turns out that this previous approach was nonsense. `path-filtering` will only run the workflow once so the variable was being set and then overwritten.  Need to instead set multiple variables and then handle in the config